source-code/app/src/main/java/org/buildmlearn/toolkit/utilities/SignerThread.java

```diff
@@ -345,7 +344,7 @@ public void attr(String ns, String name, int resourceId, int type, Object val) {
                 name = null;
             }
 
-            if (!Objects.equals(name, oldName) || val != oldVal) {
+            if ((!((name==null&&oldName==null)||(!(name==null&&oldName!=null))||(!(name!=null&&oldName==null))||(name.equals(oldName)))) || val != oldVal) {
```

Comment:
actually i am using android 4.2.2 and app is crashing when i use objects.equals....also in the editor it shows that this call requires api level 19....when i replaced this statement it's working fine....